stats/src/main/java/io/airlift/stats/TimeDistribution.java

```diff
@@ -166,6 +200,7 @@ public TimeDistributionSnapshot snapshot()
         double max;
         double[] quantiles;
         synchronized (this) {
+            DecayTDigest digest = mergeAndGetIfNeeded();
```

Comment:
should we force a merge here to avoid missing latest updates ?
i think a user of this method may expect latest state